tests/ClientTest.php

```diff
@@ -601,6 +601,15 @@ public function testImageSearch()
         $this->expectPromiseResolveWith($json, $this->client->imageSearch('clue'));
     }
 
+    public function testCommit()
+    {
+        $json = array();
+        $config = array();
+        $this->expectRequestFlow('post', 'commit?container=123&pause=1', $this->createResponseJson($json), 'expectJson');
+
+        $this->expectPromiseResolveWith($json, $this->client->commit($config, '123'));
```

Comment:
LGTM :+1: Can you add another test that checks this with all parameters set?

Reply:
Added!